journal-of-systematic-palaeontology.csl

```diff
@@ -171,6 +183,7 @@
       <key macro="author-count" names-min="3" names-use-first="3"/>
       <key macro="author" names-min="3" names-use-first="1"/>
       <key macro="year-date"/>
+     <key macro="month-date"/>
```

Comment:
Ah, yes, good catch.

@nsvitek, I agree with @adam3smith that the style guide doesn't explicitly require sorting by the full date, and it's quite likely that the current sorting is correct.